File: TOKI/Book/01-PerkenalanPemrogramanKompetitif/A.LampuDanBotol/3.Solusiterbaik/Main.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long n = sc.nextInt();

        long s = Math.round(Math.sqrt(n));

        if(s*s != n) {
            System.out.println("lampu mati");
        }else {
            System.out.println("lampu menyala");
        }
    }
}

File: TOKI/Book/01-PerkenalanPemrogramanKompetitif/A.LampuDanBotol/3.Solusiterbaik/main.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    long long N;
    cin >> N;

    long long s = round(sqrt(N));

    if(s * s != N) {
        cout << "lampu mati" << endl;
    }else {
        cout << "lampu menyala" << endl;
    }
}